JobtechSwe/myskills-web
# feat: occupation trivia

**Ticket:** [#116](https://trello.com/c/4uKKK9y1/116-yrkesinformation-info)
**Intent:**

Add trivia about chosen occupation

**How to test (optional):**

Create a CV using either systemutvecklare, lärare or florist (if you want trivia) pick  something else to make sure it works  when not having an occupation which has trivia from the API.

🔥  Depends on: https://github.com/JobtechSwe/myskills-api/pull/77 🔥 

### All of the following commands should pass.

- [ ] `npm run cypress`

### Browser testing

- [ ] Firefox
